bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -21,7 +21,3 @@ extendedArguments:
   service-cluster-ip-range: {{range .ServiceClusterIPRange}}
   - {{.}}{{end}}
   {{end}}
-  pv-recycler-pod-template-filepath-nfs: # bootstrap KCM doesn't need recycler templates
```

Comment:
why is this assumption no longer valid?  If it isn't, you'll need to render out the results.

Reply:
hm, the file isn't available during bootstrap, so it fails indeed. Removed